src/kabam/rotmg/classes/view/CharacterSkinView.as

```diff
@@ -122,7 +122,9 @@ package kabam.rotmg.classes.view
 
 		private function makeClassDetailView():ClassDetailView
 		{
-			var _local_1:ClassDetailView = new ClassDetailView();
+			var _local_1:ClassDetailView;
+			_local_1 = new ClassDetailView();
+			//var _local_1:ClassDetailView = new ClassDetailView();
```

Comment:
There is no difference how to initialize a variable. It's better to save old variant.